tests/kvlds-s3/test_kvlds.sh

```diff
@@ -10,7 +10,7 @@ TMPDIR=`pwd`/tmp
 SOCKS3=$TMPDIR/sock_s3
 SOCKL=$TMPDIR/sock_lbs
 SOCKK=$TMPDIR/sock_kvlds
-REGION=us-west-2
+REGION=${REGION:-us-west-2}
```

Comment:
I'm not certain why some REGIONs were hard-coded to `us-east-1`, while others were `us-west-2`.

Reply:
I use us-east-1 as a default, but at some point I wanted to test with a "closer" region.  I think.